bigbone/src/main/kotlin/social/bigbone/api/entity/Account.kt

```diff
@@ -134,13 +136,16 @@ data class Account(
      * When the account was created (ISO 8601 Datetime).
```

Comment:
Kdoc should no longer reference "ISO 8601 datetime" or claim that this is a potentially nullable String (for all fields that have been changed to PrecisionDateTime).